package/MDAnalysis/transformations/base.py

```diff
@@ -0,0 +1,118 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
make sure that this page shows up in docs

Reply:
Yes, under `8.5. Building blocks for Transformation Classes`.